jenkins-pipelines/rolling-upgrade-centos7.jenkinsfile

```diff
@@ -7,7 +7,7 @@ rollingUpgradePipeline(
     params: params,
 
     backend: 'gce',
-    base_versions: ['2019.1', '3.2'],
```

Comment:
This is wrong, we should have here 2019.1

Reply:
but we have

Reply:
The change is correct.

Reply:
The enterprise version will be filtered out in pipeline on opensource upgrade jobs.
Actually it doesn't effect, we can remove it.